docs/framework/wcf/extending/specifying-a-custom-crypto-algorithm.md

```diff
@@ -111,9 +111,9 @@ public class MyCustomAlgorithmSuite : SecurityAlgorithmSuite
 </configuration>  
 ```  
   
- The section under the <`cryptoClasses`> element creates the mapping between the SHA256CryptoServiceProvider and the alias "SHA256CSP". The <`nameEntry`> element creates the mapping between the "SHA256CSP" alias and the specified URL (http://constoso.com/CustomAlgorithms/CustomHashAlgorithm ).  
+ The section under the <`cryptoClasses`> element creates the mapping between the SHA256CryptoServiceProvider and the alias "SHA256CSP". The <`nameEntry`> element creates the mapping between the "SHA256CSP" alias and the specified URL (http://constoso.com/CustomAlgorithms/CustomHashAlgorithm).  
   
- To register the custom algorithm in code use the <xref:System.Security.Cryptography.CryptoConfig.AddAlgorithm%2A> System.String[])?qualifyHint=False&autoUpgrade=True method. This method creates both mappings. The following example shows how to call this method:  
+ To register the custom algorithm in code use the <xref:System.Security.Cryptography.CryptoConfig.AddAlgorithm(System.String[])> method. This method creates both mappings. The following example shows how to call this method:  
```

Comment:
The link is broken. Unless you want the parameters to appear in the link text, you should link to \<xref:System.Security.Cryptography.CryptoConfig.AddAlgorithm%2A). Otherwise, the correct xref should be \<xref:System.Security.Cryptography.CryptoConfig.AddAlgorithm(System.Type,System.String[])>.